README.md

```diff
@@ -1,9 +1,12 @@
 # Invoicer
 
-To run the server use these commands:
-
-`npm install @material/web`
-
-`npm install --save-dev @web/dev-server`
-
-`npx web-dev-server --node-resolve`
+#### The site was created to enter accounting for turnover, control the amount of products in the warehouse, calculate customer debts, the total amount of profit received, create invoices (on the site and export in PDF format, for printing)
+To clone repo use command:\
+`git clone https://github.com/student-project-lnu-2022/invocer.git` \
+Go to directory invoicer_project, open terminal and run next commands:\
+`pip install djangorestframework` \
+`pip install django-cors-headers` \
+`pip install djangorestframework-simplejwt` \
+`pip install reportlab` \
+To run the server use command:\
+`python manage.py runserver`
```

Comment:
Here you missed `makemigrations `and `migrarte `command

Reply:
done